src/checkstyle/checkstyle.xml

```diff
@@ -178,4 +178,4 @@
 		<module name="WhitespaceAround" />
 
 	</module>
-</module>
+</module>
```

Comment:
Not sure what led to make some changes in this file, but we definitely require a new line in the end of each file.